WebdriverIO/tests/response-modification.spec.ts

```diff
@@ -32,31 +36,33 @@ describe('Response modification', () => {
         expectedURL,
         expectedResponseStatusCode,
       );
-      await browser.assertExpectedRequestsOnly();
+      await browser.expectRequest(
```

Comment:
Why do you call this `expectRequest` method twice?

Reply:
They are called with different status codes: `expectedResponseStatusCode` and `modifiedResponseStatusCode`. And after that is called the `assertRequests()` which compares the expectations.

Reply:
Then my question is, why is the first one called? :D
My way of thinking is clicking only once, and validating that modified response.

Reply:
On second thought, I agree with you. I see no point in calling the first one. I modified it. Thanks :)